cmd/version.go

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2021 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+var outputAsJSON bool
+
+var VersionCmd = &cobra.Command{
+	Use:   "version",
+	Short: "Print Dapr Runtime and Cli's version.",
+	Example: `
+# Version for Dapr
+dapr version --output-as-json
+`,
+	Run: func(cmd *cobra.Command, args []string) {
+		if outputAsJSON {
+			b, err := json.Marshal(daprVer)
+			if err != nil {
+				panic(err)
+			}
+			fmt.Printf("%s", string(b))
+		} else {
+			fmt.Printf("CLI version: %s \nRuntime version: %s", daprVer.CLI, daprVer.Runtime)
+		}
+	},
+}
+
+func init() {
+	VersionCmd.Flags().BoolP("help", "h", false, "Print this help message")
+	VersionCmd.Flags().BoolVarP(&outputAsJSON, "output-as-json", "", false, "version command's output is in JSON format.")
```

Comment:
@fanfanbj Do you think having this as `--output` following other commands with JSON as one recognised value is better? I am not sure if YAML is something that we want for this command. 
Just for consistency I think `--output` flag is better with recognised inputs as JSON, TEXT, with text being default?

Reply:
@mukundansundar I changed the flag to `--output`. And I look at other commands, so I only implement JSON value.